src/community/index.md

```diff
@@ -7,6 +7,10 @@ description: "Communities, mailing lists, and bug databases for the Dart project
 
 Track the Dart project, get help, and talk with other Dart developers.
 
+We pledge to maintain an open and welcoming environment.
+For details, see our [code of conduct](/code-of-conduct).
+
+
```

Comment:
extra line?

Reply:
We often have two blank lines above headings, just to make it easier to find them. Is that a problem?

Reply:
Not if it's intentional 😄